crowbar_framework/app/views/nodes/_form.html.haml

```diff
@@ -105,6 +105,39 @@
         %span.help-block
           = t('.intended_role_hint')
 
+      .form-group
+      = f.label :disk_roles, t('.disk_roles')
+
+      %table.table.table-middle
+        %thead
+          %tr
+            %th.col-sm-2
+              = t(".disk_short")
+            %th.col-sm-3
+              = t(".disk_unique")
+            %th.col-sm-2
+              = t(".disk_role")
+        %tbody
+          - all_disks = all_disks_info
+          - (@node.crowbar_wall["claimed_disks"] || {}).each do |unique_name, disk|
+            %tr
+              %td
+                = all_disks.fetch(unique_name, unique_name)
```

Comment:
I wonder if the fallback should really be `unique_name`. Shouldn't we just put the usual long dash that is displayed when we don't have data? The issue is that we already display `unique_name` on the column after that.